design_docs/oscode/oscode_sundials.md

```diff
@@ -3,50 +3,59 @@
+Solutions to oscillatory problems are often bespoke and use analytic approximations such as the Wentzel-Kramers-Brilouin (WKB) method used by OSCODE. Giving researchers access to a general purpose solver for solutions for ODEs of the form written above will allow them to explore more computationally complex models in their research and speed up existing implementations.
 
-The second order ODE can be reposed as a first order ODE to fit into the standard schema for sundials solvers
+The second order ODE can be reposed as a set of two, first order ODEs to fit into the standard schema for sundials solvers,
 
 $$
 \dot{y} = \begin{pmatrix}
 y[1]\\
 -y[0]w^2(t)-2\gamma(t)y[1]
-\end{pmatrix}
+\end{pmatrix}.
 $$
 
-The most similar module is ARKODE, which solves equations with known explicit and implicit nonstiff and stiff time scale components. 
+The most similar module is ARKODE, which solves equations with known explicit and implicit, nonstiff and stiff time scale components, 
 
 $$
-M(t)\ddot{y} = f^E(t, y) + f^I(t, y), y(t_0) = y_0
+M(t)\ddot{y} = f^E(t, y) + f^I(t, y), \quad y(t_0) = y_0.
 $$
 
-For the new OSCODE module we propose the equation will take the form
+<!--- Are you sure the \ddot above is correct? I'd expect just \dot.--->
+
+For the new OSCODE module we propose the right-hand-side will be a subset of what ARKODE can handle, specifically,
 
 $$
-M(t)\ddot{y} = f^B(t, y), y(t_0) = y_0
+M(t)\ddot{y} = f^B(t, y), \quad y(t_0) = y_0
 $$
 
+<!--- What's the B in the above? --->
+
+with $f^B(t, y)$ taking the form as the right-hand-side of a second-order, linear, homogeneous ODE as described above. OSCODE will then dynamically switch between a Runge-Kutta solver and its asymptotic WKB-based solver, at each step choosing the method which yields the larger step length (while keeping the local error within user tolerance).
 
-where $$f^B(t, y)$$ has the ability to switch dynamically between a Runge-Kutta (RK) solver and the Wentzel-Kramers-Brillouin (WKB) solver based on the relative error of the RK and WKB methods.
+The OSCODE framework is packaged with a built in Butcher tableau for a 6-stage, 5th order, and 4-stage, 4th order Runge-Kutta method, though users can supply their own Buthcer tableaus of any order. 
+
+<!--- OSCODE also supports unevenly spaced samples of $y$. What do you mean by this? --->
 
-OSCODE's ability to switch between the RK and WKB solvers allow for accurate and easy time integration of mixed stiff/nonstiff systems of ordinary differential equations. The OSCODE framework is packaged with built in butcher tables for order 6, though users can supply their own buther tables of any order. OSCODE also supports unevenly spaced samples of $y$.
 
 Runge-Kutta Solver:
 
 
 Wentzel-Kramers-Brillouin Solver:
 
+<!--- What do you want me to put here? A quick math overview of each? --->
```

Comment:
Nope! I was just going to either delete these or put a small explainer of the two solvers that are in the code. But I'm not sure that is necessary. Maybe for the WKB solver?